cgk_calcium_tools/filehandler.py

```diff
@@ -1498,6 +1318,10 @@ def preprocess_step(
     None
 
     """
+    
+    # Initialize progress bar
+    pb = progress_bar(amount_of_files, 'Preprocessing')
```

Comment:
amount_of_files is just len(pairlist[0]), or do you plan to use  it in another way?